sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/aio/_backup_client.py

```diff
@@ -0,0 +1,104 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+from typing import TYPE_CHECKING
+
+from azure.core.polling.async_base_polling import AsyncLROBasePolling
+
+from .._internal import AsyncKeyVaultClientBase
+from .._internal.polling import KeyVaultBackupClientPolling
+from .._models import BackupOperation, RestoreOperation, SelectiveKeyRestoreOperation
+
+if TYPE_CHECKING:
+    # pylint:disable=unused-import
+    from typing import Any
+    from azure.core.polling import AsyncLROPoller
+
+
+class KeyVaultBackupClient(AsyncKeyVaultClientBase):
+    """Performs Key Vault backup and restore operations.
+
+    :param str vault_url: URL of the vault on which the client will operate. This is also called the vault's "DNS Name".
+    :param credential: an object which can provide an access token for the vault, such as a credential from
+        :mod:`azure.identity.aio`
+    """
+
+    # pylint:disable=protected-access
+    async def begin_full_backup(
+        self, blob_storage_uri: str, sas_token: str, **kwargs: "Any"
+    ) -> "AsyncLROPoller[BackupOperation]":
+        """Begin a full backup of the Key Vault.
+
+        :param str blob_storage_uri: URI of the blob storage resource in which the backup will be stored
+        :param str sas_token: a Shared Access Signature (SAS) token authorizing access to the blob storage resource
+        :keyword str continuation_token: a continuation token to restart polling from a saved state
+        :returns: An AsyncLROPoller. Call `result()` on this object to get a :class:`BackupOperation`.
+        :rtype: ~azure.core.polling.AsyncLROPoller[BackupOperation]
+        """
+        polling_interval = kwargs.pop("_polling_interval", 5)
+        sas_parameter = self._models.SASTokenParameter(storage_resource_uri=blob_storage_uri, token=sas_token)
+        return await self._client.begin_full_backup(
+            vault_base_url=self._vault_url,
+            azure_storage_blob_container_uri=sas_parameter,
+            cls=BackupOperation._wrap_generated,
+            continuation_token=kwargs.pop("continuation_token", None),
+            polling=AsyncLROBasePolling(
+                lro_algorithms=[KeyVaultBackupClientPolling()], timeout=polling_interval, **kwargs
+            ),
+            **kwargs
+        )
+
+    async def begin_full_restore(
+        self, blob_storage_uri: str, sas_token: str, folder_name: str, **kwargs: "Any"
+    ) -> "AsyncLROPoller[RestoreOperation]":
+        """Restore a Key Vault backup.
```

Comment:
nit per the comment on [the sync version](https://github.com/Azure/azure-sdk-for-python/pull/13525/files#r482486160)
```suggestion
        """Restore a full backup of a Key Vault.
```